ui/admin/tests/acceptance/aliases/create-test.js

```diff
@@ -113,13 +106,13 @@ module('Acceptance | aliases | create', function (hooks) {
       );
     });
     await visit(urls.newAlias);
-    await click(SAVE_BTN_SELECTOR);
+    await click(commonSelectors.SAVE_BTN);
     await a11yAudit();
 
     assert
       .dom(commonSelectors.ALERT_TOAST_BODY)
       .hasText('The request was invalid.');
-    assert.dom(FIELD_ERROR_TEXT_SELECTOR).hasText('Name is required.');
+    assert.dom(commonSelectors.FIELD_TEXT_ERROR).hasText('Name is required.');
```

Comment:
I know the code was already there, as part of refactoring selectors, and one of the reasons to keep it one domain at a time, is to also "evaluate" our tests.

This test is mocking a field error and asserting the form behaves the way we expect. In this case, we are marking the field name as required, but the aliases creation form does not have the field name as required.
The aliases creation form, has as required alias value. I will suggest to modify this mock to mark as required the alias value field.
And as a selector for such error message [I will recomend to follow what has been mentioned here](https://github.com/hashicorp/boundary-ui/pull/2685#discussion_r1962277721).